lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+  @override
+  _EditTileScreenState createState() => _EditTileScreenState();
+}
+
+class _EditTileScreenState extends State<EditTileScreen> {
+  ScrollController _scrollController;
+  // double _scrollOffset = 0.0;
+  bool lock = false;
+
+  @override
+  void initState() {
+    _scrollController = ScrollController()
+      ..addListener(() {
+        // setState(() {
+        //   _scrollOffset = _scrollController.offset;
+        // });
+      });
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _scrollController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+
+    return Scaffold(
+      appBar: PreferredSize(
+        preferredSize: Size.fromHeight(screenSize.height) / 15,
+        child: AppBar(
+          leading: GestureDetector(
+            onTap: () => {
+              Navigator.push(context,
+                  MaterialPageRoute(builder: (context) => UnlockedHomeScreen()))
+            },
+            child: Icon(Icons.arrow_back, color: Colors.white, size: 25.0),
+          ),
+          centerTitle: true,
+          title: Text(
+              'Edit'), // Debug: Change title to keep track of how many tiles
+        ),
+      ),
+      body: Container(
+        child: Column(
+          children: <Widget>[
+            SizedBox(
+              height: screenSize.height / 75,
+            ),
+            Stack(
+              children: <Widget>[
+                lock
+                    ? Positioned(
+                        left: 10.0,
+                        child: Icon(
+                          Icons.lock,
+                          color: Colors.black45,
+                          size: 75,
+                        ),
+                      )
+                    : SizedBox(width: 0, height: 0),
+                Center(
+                  child: Container(
+                    width: screenSize.width / 2,
+                    height: 168,
+                    child: ClipRRect(
+                      borderRadius: BorderRadius.circular(8.0),
+                      child: Card(
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          crossAxisAlignment: CrossAxisAlignment.center,
+                          children: <Widget>[
+                            Expanded(
+                              child: Container(
+                                color: Color(0xffF8C199),
+                                child: GestureDetector(
+                                  onTap: () => {
+                                    showModalBottomSheet(
+                                      context: context,
+                                      builder: (context) {
+                                        return Wrap(
+                                          children: [
+                                            ListTile(
```

Comment:
If you want, you can also turn this into an array to avoid repeating some code here! Take a look at @nganphan123's branch to see some of the cool work that she did!